### DISCUSSION TITLE
Add multiple Parent fileClass to a tag

### DISCUSSION THREAD

**Discussion by vkbytes** [mdelobelle/metadatamenu#272] [General]
In the Fileclass settings we can add a `Parent fileclass` as a extended keys. But this allows only one file to be added, can we have a array of `Parent fileclass` that a tag can extend its keys from ?

**Comment by lucas-wu-wakakak** [comment]
besides, parrent fileclass table view cannot list children fileclass caller's same properties data. It would be great if both problems are solved.
